src/Compilers/VisualBasic/Test/Syntax/IncrementalParser/IncrementalParser.vb

```diff
@@ -2602,7 +2602,7 @@ End Class
             Dim c = str(i)
             If (c = vbLf(0)) AndAlso
                 ((i = str.Length - 1) OrElse (str(i + 1) <> vbCr(0))) Then
-                builder.AddRange(vbCrLf)
+                builder.AddRange(DirectCast(vbCrLf, IEnumerable(Of Char)))
```

Comment:
Why this change?

Reply:
Oh, this one can be reverted, thanks. I had added an ambiguous overload to `ArrayBuilder` with `ReadOnlySpan`, but had to revert it.

---
In reply to: [609090644](https://github.com/dotnet/roslyn/pull/52422#discussion_r609090644) [](ancestors = 609090644)